src/entity/builtin_entity.rs

```diff
@@ -656,6 +780,18 @@ impl BuiltinEntityKind {
                 Language::PT_BR,
                 Language::PT_PT,
             ],
+            BuiltinEntityKind::Date => &[
+                Language::EN,
+            ],
+            BuiltinEntityKind::Time => &[
+                Language::EN,
+            ],
+            BuiltinEntityKind::DatePeriod => &[
+                Language::EN,
+            ],
+            BuiltinEntityKind::TimePeriod => &[
+                Language::EN,
+            ],
```

Comment:
Code style detail: you can put short lists on a single line for compactness.

Reply:
✅